static/app/actionCreators/pageFilters.tsx

```diff
@@ -323,15 +323,15 @@ export function updateDateTime(
 ) {
   PageFiltersActions.updateDateTime(datetime);
   updateParams(datetime, router, options);
-  persistPageFilters(options);
+  persistPageFilters('datetime', options);
 }
 
 /**
  * Pins a particular filter so that it is read out of local storage
  */
 export function pinFilter(filter: PinnedPageFilter, pin: boolean) {
   PageFiltersActions.pin(filter, pin);
-  persistPageFilters({save: true});
+  persistPageFilters(null, {save: true});
```

Comment:
is this call still necessary? what changes would this make to local storage

Reply:
it persists what is pinned. The comment above `persistPageFilters notes that it will always persist the pin state

Reply:
ahhh i see makes sense